content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,44 +7,195 @@ description: >
+
+The physical network approach offers you several advantages:
+
+* Common physical network to handle K8s and non-K8s traffic.
+* Optimal network performance.
+* Enhanced security with a CNI plugin that handles encrypted communications such as IPsec or TLS.
+
+The challenges to you include CNI plugin management and non-K8s network operations impact to your cluster.
+
+For more information on two examples of physical network implementations, see [Calico BGP](https://github.com/projectcalico/calico) and [Cilium native-routing](https://docs.cilium.io/en/v1.13/network/concepts/routing/#native-routing).
+
+#### Pods on different nodes using a virtual overlay network
+
+You can address your inter-node pod networking requirements by deploying a [virtual overlay network](https://book.systemsapproach.org/applications/overlays.html). This approach employs virtual tunnels to form a separate logical _overlay network_ for the express purpose of transporting packets from one pod to another.
+
+Figure 6 illustrates the notion of a virtual overlay network running through (or on top of) a physical underlay network.   
+
+{{< figure src="/docs/images/k8s-net-vert-overlay-big2.drawio.svg" alt="diagram of virtual overlay network running on top of a physical underlay network." class="diagram-large" caption="Figure 6. Example of a virtual overlay network" >}}
+ 
+Pods 1, 2 and 3 are networked together through a mesh of virtual tunnels. Your encapsulated pod packets injected into a virtual tunnel are transported from one node to another across your physical underlay network. 
+
+{{< note >}}
+A physical network is sometimes referred to as a _physical underlay network_. This because it is transporting virtual overlay network packets.   
+{{< /note >}}
+
+Figure 7 shows an example of a virtual overlay network composed of two pods running on separate nodes and connected via virtual overlay network tunnel. It uses pod and root network namespaces, network plugins, tunnel encapsuation/decapsulation, and a physical underlay network.
+ 
+{{< figure src="/docs/images/k8s-net-virtual-overlay.drawio.svg" alt="diagram of 2 x k8s pods on separate nodes using virtual tunnels" class="diagram-large" caption="Figure 7. Example of Pod 1 - Pod 2 networking on different nodes using a virtual overlay network tunnel" >}}
+
+In this example, the network plugin has set up a _virtual overlay network_. Pod packets sourced on one node and destined for Pods 
+on a different node are encapsulated and sent over a tunnel mechanism such as [GENEVE](https://en.wikipedia.org/wiki/Generic_Network_Virtualization_Encapsulation). 
+
+The destination node retrieves the encapsulated pod packet, strips off the encap header, and then sends it to the target Pod, providing that there are no
+[NetworkPolicy](/docs/concepts/services-networking/network-policies/) restrictions to block that packet.
```

Comment:
For this example, consider sending the VPN traffic over a second host interface. I'd like to make the point that nodes can have more than one interface.

Reply:
Got it!